web/src/main/resources/datainfo.properties

```diff
@@ -364,4 +364,15 @@ pformMaxSubmissionSize=100000000
 form.engine.url=http://yogi.local:8006
 ocform.adminapikey=ocrocks
 SBSUrl=http://yogi.dev-sbs.openclinica.info:8080/user-service/api/users/
```

Comment:
This URL should be updated to the production value.  They are developer specific at the moment.